src/libs/cropOrRotateImage/index.native.ts

```diff
@@ -0,0 +1,18 @@
+import RNImageManipulator from '@oguzhnatly/react-native-image-manipulator';
+import RNFetchBlob from 'react-native-blob-util';
+import {CropOrRotateImage} from './types';
+
+/**
+ * Crops and rotates the image on ios/android
+ */
+const cropOrRotateImage: CropOrRotateImage = (uri, actions, options) =>
+    new Promise((resolve) => {
+        RNImageManipulator.manipulate(uri, actions, options).then((result) => {
+            RNFetchBlob.fs.stat(result.uri.replace('file://', '')).then(({size}) => {
+                const file = Object.assign(result, {size, type: options.type || 'image/jpeg', name: options.name || 'fileName.jpg'});
+                resolve(file);
```

Comment:
```suggestion
                resolve({
                    ...result,
                    size,
                    type: options.type || 'image/jpeg',
                    name: options.name || 'fileName.jpg',
                });
```

The old way still works, isn't?

Reply:
Yeah, fixed @fabioh8010